LTP changes for sgx-lkl #236 being fixed

SeanTAllen opened the pull request:
Closes #14.

SeanTAllen commented:
This should not be merged until https://github.com/lsds/sgx-lkl/pull/723 is merged.

hukoyu reviewed (approved):
It looks good. The comments we put made it easier to fix back I assume :) Thanks @SeanTAllen fixing this bug end fixing the related LTP tests.

Commit:
Undo patches related to sgx-lkl #236

This undoes the changes associated with the bug as seen in
https://github.com/lsds/sgx-lkl/issues/236.

Closes #14
